config.toml

```diff
@@ -102,6 +102,10 @@ min_k8s_version = "v1.20.0"
 [[params.versions]]
   version = "v0.8"
   url = "https://release-v0-8.docs.openservicemesh.io/"
+[params.versionbanner]
+	show = false
+	archive = "v1.0"
```

Comment:
v1.1 is the latest version; wouldn't we still want the banner shown for v1.0?

Reply:
Thanks @keithmattix 

The previous banner should only be enabled on release branches of previous versions, eg:
- https://github.com/openservicemesh/osm-docs/pull/398
- https://github.com/openservicemesh/osm-docs/pull/397
- https://github.com/openservicemesh/osm-docs/pull/396
- https://github.com/openservicemesh/osm-docs/pull/395
- https://github.com/openservicemesh/osm-docs/pull/394

The current release branch and main should always have the banner disabled.

Reply:
Pausing on this one based on the conversation around how many old versions to keep docs for.